va/drm/va_drm_utils.c

```diff
@@ -41,8 +41,8 @@ static const struct driver_name_map g_driver_name_map[] = {
     { "pvrsrvkm",   8, "pvr"    }, // Intel UMG PVR driver
     { "emgd",       4, "emgd"   }, // Intel ECG PVR driver
     { "hybrid",     6, "hybrid" }, // Intel OTC Hybrid driver
-    { "nouveau",    7, "gallium" }, // Mesa Gallium driver
-    { "radeon",     6, "gallium" }, // Mesa Gallium driver
+    { "nouveau",    7, "nouveau"  }, // Mesa Gallium driver
+    { "radeon",     6, "r600"     }, // Mesa Gallium driver
```

Comment:
Yes that was my earlier concern.  I'm leery of chasing a Mesa change when we break existing users.  I'd also like Julien to weigh in on this since he originally requested this change. @julien.isorce@gmail.com